lib/index.js

```diff
@@ -185,52 +185,9 @@ function PubSubClient(opts) {
 				self.emit('error', e);
 			}
 		});
-		// on a requeue, we are going to send a custom metric to newrelic if configured
-		this.on('connect_error', function (err) {
-			try {
-				newrelic.noticeError(err, {
-					key: opts.key,
-					url: self.url,
-					fingerprint: fingerprint
-				});
-			} catch(e) {
-				self.emit('error', e);
-			}
-		});
 	}
 	this.queueSendDelay = opts.queueSendDelay === undefined ? 10 : Math.max(1, +opts.queueSendDelay);
```

Comment:
@RobCatch let's also remove queuing and delayed send (and related conf and methods). Prefer to just fire immediately with retry.